nautobot/utilities/tables.py

```diff
@@ -417,7 +417,7 @@ def __init__(self, relationship, side, *args, **kwargs):
     def render(self, record, value):  # pylint: disable=arguments-differ
         if value is None:
             # This returns None if value is None
-            return self.default
+            return "—"
```

Comment:
So I did some digging and found out that `self.default` has always be None. The trick is the `empty_values` attribute of the column. The docstring of [render()](https://github.com/jieter/django-tables2/blob/1d0dfa7d9ad209332fb1f6936d213c4363455075/django_tables2/columns/base.py#L357) said that if the value returned is in `empty_values`, `render()` method will be skipped. So any empty or null value handling in the `render()` method itself is essentially not doing anything. We have to specify the empty_values attribute to include all possible invalid values for — to be rendered. The relationship column didnt work because the default `empty_values` is `(None, "")` whereas a relationship can return `[]`as an empty value, so we have to include `[]`.

Reply:
Same thing CustomFieldColumn, so made that change as well.